autozone-poc/src/components/Home.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import BgImg from '../utlis/img/bg-img/bg-1.jpg';
+import BgImgTopCatArea1 from '../utlis/img/bg-img/bg-2.jpg';
+import BgImgTopCatArea2 from '../utlis/img/bg-img/bg-3.jpg';
+import BgImgTopCatArea3 from '../utlis/img/bg-img/bg-4.jpg';
+import BgImgCatArea from '../utlis/img/bg-img/bg-5.jpg';
+import PropTypes from 'prop-types';
+import Button from '../utlis/button/Button';
+import TopCatagoryArea from '../utlis/top_catagory_area/TopCatagoryArea';
+import SimpleSlider from '../utlis/slick_slider/SimpleSlider';
+import ScrollUpArrowBtn from '../utlis/button/ScrollUpArrowBtn';
+
+
+
+const Home = props => {
+  const bgImgStyle = {
+    'backgroundImage': `url(${BgImg})`
+  }
+  const bgImgTopCatArea1 = {
+    'backgroundImage': `url(${BgImgTopCatArea1})`
+  }
+  const bgImgTopCatArea2 = {
+    'backgroundImage': `url(${BgImgTopCatArea2})`
+  }
+  const bgImgTopCatArea3 = {
+    'backgroundImage': `url(${BgImgTopCatArea3})`
+  }
+  const bgImgCatArea = {
+    'backgroundImage': `url(${BgImgCatArea})`
+  }
+
+    return (
+      <div>
+        <ScrollUpArrowBtn/>
+        <section className="welcome_area bg-img" style={bgImgStyle}>
+          <div className="container h-100">
+            <div className="row h-100 align-items-center">
+              <div className="col-12">
+                <div className="hero-content">
+                  <h6>asoss</h6>
+                  <h2>New Collection</h2>
+                  <Button btnText="view collection" />
+                </div>
+              </div>
+            </div>
+          </div>
+        </section>
+        <div className="top_catagory_area section-padding-80 clearfix">
+          <div className="container">
+            <div className="row justify-content-center">
+              <TopCatagoryArea topCatagoryAreaText="Clothing" style={bgImgTopCatArea1} />
```

Comment:
instead of using multiple time please make it single time usage and pass the value as props

Reply:
Changed into single by using props